<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dynamicadd</class>
 <widget class="QDialog" name="Dynamicadd">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Consolas</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>添加</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icon/icon/add.png</normaloff>:/icon/icon/add.png</iconset>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>578</width>
     <height>381</height>
    </rect>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <widget class="QWidget" name="tabline">
    <property name="font">
     <font>
      <family>Consolas</family>
     </font>
    </property>
    <attribute name="icon">
     <iconset>
      <normaloff>:/icon/icon/subway.png</normaloff>:/icon/icon/subway.png</iconset>
    </attribute>
    <attribute name="title">
     <string>线路</string>
    </attribute>
    <widget class="QLabel" name="linelabel">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>11</y>
       <width>551</width>
       <height>178</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;动态添加方式：&lt;/span&gt;&lt;/p&gt;&lt;p&gt;1.&lt;/p&gt;&lt;p&gt;1)添加线路，包括路线名称和路线颜色&lt;/p&gt;&lt;p&gt;2)添加站点，包括站点名称、位置和所属路线&lt;/p&gt;&lt;p&gt;3)添加连接，包括相接的站点和所属路线&lt;/p&gt;&lt;p&gt;2.文本方式添加&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonline">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>300</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
      </font>
     </property>
     <property name="text">
      <string>添加线路</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QWidget" name="formLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>200</y>
       <width>261</width>
       <height>81</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="linelabel_1">
        <property name="text">
         <string>线路名称：</string>
        </property>
        <property name="buddy">
         <cstring>lineEditlinename</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditlinename">
        <property name="placeholderText">
         <string>请输入线路名</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="linelabel_2">
        <property name="text">
         <string>线路颜色：</string>
        </property>
        <property name="buddy">
         <cstring>pushButtoncolor</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButtoncolor">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">AA</string>
        </property>
        <property name="text">
         <string>选择颜色</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tabstation">
    <attribute name="icon">
     <iconset>
      <normaloff>:/icon/icon/station.png</normaloff>:/icon/icon/station.png</iconset>
    </attribute>
    <attribute name="title">
     <string>站点</string>
    </attribute>
    <widget class="QPushButton" name="pushButtonstation">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>310</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
      </font>
     </property>
     <property name="text">
      <string>添加站点</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>50</y>
       <width>221</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="stationlabel_6">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>20</y>
       <width>90</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>选择所属线路</string>
     </property>
    </widget>
    <widget class="QWidget" name="formLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>281</width>
       <height>191</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_4">
      <item row="1" column="0">
       <widget class="QLabel" name="stationlabel_1">
        <property name="text">
         <string>站点名称：</string>
        </property>
        <property name="buddy">
         <cstring>lineEditstationname</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditstationname">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>添加站点名</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="stationlabel_3">
        <property name="minimumSize">
         <size>
          <width>35</width>
          <height>20</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
         </font>
        </property>
        <property name="text">
         <string>坐标东经(121-122):</string>
        </property>
        <property name="buddy">
         <cstring>doubleSpinBoxlongitude</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBoxlongitude">
        <property name="minimumSize">
         <size>
          <width>85</width>
          <height>20</height>
         </size>
        </property>
        <property name="decimals">
         <number>7</number>
        </property>
        <property name="minimum">
         <double>121.000000000000000</double>
        </property>
        <property name="maximum">
         <double>122.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>121.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="stationlabel_4">
        <property name="minimumSize">
         <size>
          <width>35</width>
          <height>20</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
         </font>
        </property>
        <property name="text">
         <string>坐标北纬(30.9-31.5):</string>
        </property>
        <property name="buddy">
         <cstring>doubleSpinBoxlatitude</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBoxlatitude">
        <property name="minimumSize">
         <size>
          <width>85</width>
          <height>20</height>
         </size>
        </property>
        <property name="decimals">
         <number>7</number>
        </property>
        <property name="minimum">
         <double>30.899999999999999</double>
        </property>
        <property name="maximum">
         <double>31.449999999999999</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>30.899999999999999</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tabconnect">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <attribute name="icon">
     <iconset>
      <normaloff>:/icon/icon/connect.png</normaloff>:/icon/icon/connect.png</iconset>
    </attribute>
    <attribute name="title">
     <string>连接</string>
    </attribute>
    <widget class="QLabel" name="connectlabel_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>560</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;连接站点提示:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;1.站点及所属线路选择可以选择也可直接输入&lt;/p&gt;&lt;p&gt;2.“站点1”和“站点2”必须为同一线路中的不同站点&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>160</y>
       <width>291</width>
       <height>111</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="connectlabel_2">
        <property name="text">
         <string>站点1：</string>
        </property>
        <property name="buddy">
         <cstring>comboBoxstation1</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxstation1">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
         </font>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="connectlabel_3">
        <property name="text">
         <string>站点2：</string>
        </property>
        <property name="buddy">
         <cstring>comboBoxstation2</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBoxstation2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
         </font>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="connectlabel_4">
        <property name="text">
         <string>所属线路：</string>
        </property>
        <property name="buddy">
         <cstring>comboBoxconnectline</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBoxconnectline">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
         </font>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="pushButtonconnect">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>290</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
      </font>
     </property>
     <property name="text">
      <string>连接</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabtext">
    <attribute name="title">
     <string>文本</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0" colspan="3">
      <widget class="QTextEdit" name="textEdit"/>
     </item>
     <item row="1" column="0">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>219</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButtontext">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
        </font>
       </property>
       <property name="text">
        <string>添加</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>218</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
